src/AddTodoForm.js

```diff
@@ -1,14 +1,38 @@
-import React from "react";
-
-function AddTodoForm() {
-    return (
-      <form>
-        <label htmlFor="todoTitle">Title</label>
-        <input type="text" id="todoTitle" />
-        <button type="submit">Add</button>
-      </form>
-    );
-  }
-  
+import React, { useState } from "react";
+
+function AddTodoForm({ onAddTodo }) {
+  // Create a new state variable named todoTitle with an update function named setTodoTitle
+  const [todoTitle, setTodoTitle] = useState("");
```

Comment:
This state is unnecessary as everything can be handled fine without it and it wasn't in the assignment

Reply:
if i remove the state, I get errors in my code